meshroom/nodes/aliceVision/SketchfabUpload.py

```diff
@@ -0,0 +1,200 @@
+        self._cb_kwargs.update({
+            'size'    : self._len,
+            'progress': self._progress
+        })
+        if self._callback:
+            try:
+                self._callback(*self._cb_args, **self._cb_kwargs)
+            except Exception as e: # catches exception from the callback
+                self._cb_kwargs['logManager'].logger.warning('Error at callback: {}'.format(e))
+        return chunk
+
+def progressUpdate(size=None, progress=None, logManager=None):
+    if not logManager.progressBar:
+        logManager.makeProgressBar(size, 'Upload progress:')
+
+    logManager.updateProgressBar(progress)
+
+class SketchfabUpload(desc.Node):
+    size = desc.DynamicNodeSize('inputFiles')
+    inputs = [
+        desc.ListAttribute(
+            elementDesc=desc.File(
+                name="input",
+                label="Input",
+                description="",
+                value="",
+                uid=[0],
+            ),
+            name="inputFiles",
+            label="Input Files",
+            description="Input Files to export.",
+            group="",
+        ),
+        desc.ChoiceParam(
+            name='maxSize',
+            label='Maximum Upload Size',
+            description='The maximum upload size in MB.',
+            value=50,
+            values=(50, 200, 500),
+            exclusive=True,
+            uid=[0],
+            advanced=True,
+        ),
+        desc.BoolParam(
+            name='limitSize',
+            label='Limit Upload Size',
+            description='Change maximum download size in advanced attributes.',
+            value=True,
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='apiToken',
+            label='API Token',
+            description='Get your token from https://sketchfab.com/settings/password',
+            value='',
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='title',
+            label='Title',
+            description='Title cannot be longer than 48 characters.',
+            value='',
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='description',
+            label='Description',
+            description='Description cannot be longer than 1024 characters.',
+            value='',
+            uid=[0],
+        ),
+        desc.StringParam(
+            name='license',
+            label='License',
+            description='License label.',
+            value='CC Attribution',
+            uid=[0],
+        ),
+        desc.BoolParam(
+            name='isPublished',
+            label='Publish',
+            description='If the model is not published it will be saved as a draft.',
+            value=False,
+            uid=[0],
+        ),
+        desc.BoolParam(
+            name='isInspectable',
+            label='Inspectable',
+            description='Allow 2D view in model inspector.',
+            value=True,
+            uid=[0],
+        ),
```

Comment:
+ Private Model: OFF (default) and Age-Restricted content: OFF (default) 
to complete the model properties